app/javascript/packages/document-capture/components/in-person-location-post-office-search-step.tsx

```diff
@@ -69,16 +69,22 @@ function InPersonLocationPostOfficeSearchStep({ onChange, toPreviousStep, regist
           json: selected,
           method: 'PUT',
         });
+        // In try block set success of request. If the request is successful, fire remaining code?
         if (mountedRef.current) {
           setAutoSubmit(true);
           setImmediate(() => {
-            // continue with navigation
             e.target.disabled = false;
-            e.target.click();
+            trackEvent('IdV: location submitted', {
+              selected_location: selectedLocationAddress,
+              in_person_cta_variant: inPersonCtaVariantActive,
+            });
+            forceRedirect(inPersonURL!);
```

Comment:
What's the behavior of these `trackEvent` and `forceRedirect` calls during hybrid flow?

Reply:
Good q. So trackEvent could get called twice, once on the typical submission event and then here at line 77. And then the forceRedirect behavior is probably being overridden.

Reply:
Options: I could wrap the trackEvent and forceRedirect in an if check analogous to what I do for the event.preventDefault; or I could try to take advantage of the e.click() behavior that pre-existed my changes.

Reply:
I opted for wrapping the forceRedirect trackEvent calls in an if check @NavaTim